doc/query_dsl.md

```diff
@@ -1,15 +1,416 @@
-# Commando Query DSL
+<div align="center">
+<h1>Query DSL</h1>
+</div>
 
-This mechanism positions itself as a built-in, lightweight, and somewhat limited alternative to GraphQL or Pathom3 (Pathom3 is a data query and transformation library for Clojure). Commando DSL is much more primitive and requires you to describe dependency resolution yourself. Let's look at how it works.
+The Commando Query DSL is a built-in, lightweight query mechanism. It serves as a simple alternative to more comprehensive solutions like GraphQL or Pathom3, but it is more primitive and requires you to define dependency resolution manually.
```

Comment:
I would not call it primitive, I would say it's more simple or low-level and gives more customization.